sample-apps/react-native/expo-video-sample/package.json

```diff
@@ -44,7 +44,7 @@
     "@rnx-kit/metro-resolver-symlinks": "^0.1.34",
     "@types/react": "~18.2.45",
     "@types/react-native-incall-manager": "^4",
-    "typescript": "^5.5.2"
+    "typescript": "~5.3.3"
```

Comment:
Any reason for the downgrade?

Reply:
Unintentional automatic change. Reverted with this commit https://github.com/GetStream/stream-video-js/pull/1518/commits/1d610fca867d1f905f84e9487c8c3f3bc46b3d95